src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Figure implements Drawable {

```

Comment:
```suggestion
public class Figure implements Drawable, AreaCalculator {
```

![image](https://github.com/mate-academy/jv-oop-advanced/assets/20326858/0f47ee3b-312b-478e-b92b-b3ee3cebdbcd)